crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -1,6 +1,26 @@
+import glob
+import random
+
+
+def read_names():
+    """
+    Read the names in ../names
```

Comment:
Great job using a docstring!  I love it.

to make it pep8 compliant put an empty line after the initial description

Also, conforming to a numpy style is often good.  In that case we often find it good to describe variables as "the something or rather thing".  So here the description of what is returned would be "the list of names".  I am not sure why, but this is much more readable.  I didn't make it up btw....but it really seems to work.